oc-chef-pedant/lib/pedant/rspec/search_util.rb

```diff
@@ -323,6 +323,25 @@ def can_perform_basic_searches_for(object_type)
           end
         end # can_perform_basic_searches_for
 
+        # Search for limited rows to see if we get the correct number back
```

Comment:
Let's also include a test that shows (for example) if we have three expected results, our search returns them paged correctly (2, then 1).